src/__tests__/tigris.utility.spec.ts

```diff
@@ -55,4 +57,17 @@ describe("utility tests", () => {
 		const serializedFields = Utility.facetQueryToString(facetFields);
 		expect(serializedFields).toBe(Utility.facetQueryToString(fieldOptions));
 	});
+
+	it("serializes empty sort order", () => {
+		expect(Utility.sortOrderingToString([])).toBe("[]");
+	});
+
+	it("serializes sort orders to string", () => {
+		const ordering: Ordering = [
```

Comment:
what if user constructs 

```
		const ordering: Ordering = [
			{field: "field_1", order: SortOrder.ASC},
			{field: "field_1", order: SortOrder.DESC},
];
```

maybe something to validate at later point.

Reply:
should we also leverage field name instead of string like

```
[
 field_1: SortOrder.ASC,
 field_2: SortOrder.DESC,
];
```

This will make it less error prone. Not everywhere we have this luxury but here we do. not something to do now but maybe for future improvement.